vuepress/docs/next/tutorials/devops/installation/kubernetes/kubernetes-install.md

```diff
@@ -0,0 +1,47 @@
+# Install Entando on Kubernetes
+
+## Prerequisites
+
+### Infrastructure Requirements
+
+To install Entando directly on a Kubernetes (K8s) cluster, these are the minimum infrastructure requirements:
+
+- A working DNS server configured to resolve internal and external names. If you choose to configure K8s with a global wildcard domain name for the entire cluster, be sure it points to the public IP address of the external Load Balancer (LB) of the worker nodes.
+- A default [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is needed for the Entando operator to create the correct Persistent Volume Claims (PVC) for each pod.
+- Nginx [Ingress-Controller](https://kubernetes.github.io/ingress-nginx/deploy/) is needed to expose the Entando applications.
+
+### Minimum Hardware Requirements
+
+The minimum hardware requirements depends on how you intend to use Entando 6. The more functionality you need such as a content management system (CMS), the more you will need.
+#### For Entando install only:
+
+- 6 vCPU or virtual central processing unit  (6000 millicores)
+- 6GB of RAM
+- 5GB of allocated Storage distributed in many PVCs
+
+Configure a single worker node with the requirements above or designate 3 worker nodes with the minimum of 2VPC and 4GB of RAM each.
+
+#### For Entando + Microservices:
+
+- 500 millicores per Microservice or 
```

Comment:
Should "Microservice" be capitalized? Comma after it?

Reply:
don't think you need comma for two "or" comparisons, but will check

Reply:
i'm not sure either! one of those things that probably doesn't matter as long as it's done consistently

Reply:
I think microservice is usually one word and lowercase (although sometimes MS for abbreviation), micro frontend (even though MFE is the abbreviation) is also lowercase and two words. Plenty of examples here: https://dev.entando.org/v6.3.2/tutorials/micro-frontends/react.html#test-micro-frontend. We should probably stick to that.